conf/routes

```diff
@@ -4,10 +4,15 @@
 # ~~~~
 
 # An example controller showing a sample home page
-GET     /                           controllers.HomeController.index()
+GET     /                               controllers.HomeController.index()
 
 # Map static resources from the /public folder to the /assets URL path
-GET     /assets/*file               controllers.Assets.versioned(path="/public", file: Asset)
+GET     /assets/*file                   controllers.Assets.versioned(path="/public", file: Asset)
 
 # API endpoints
-GET /pass-bakery/status             controllers.APIController.serviceStatus()
+GET /pass-bakery/status                 controllers.APIController.serviceStatus()
+GET /rest/bakery/product/:id            controllers.APIController.findByID(id: java.util.UUID)
+PUT /rest/bakery/product/:id            controllers.APIController.editByID(id: java.util.UUID)
+DELETE /rest/bakery/product/:id         controllers.APIController.deleteByID(id: java.util.UUID)
+GET /rest/bakery/product                controllers.APIController.findAllProducts()
+POST /rest/bakery/product               controllers.APIController.postProduct()
```

Comment:
Indent the endpoint paths as well and group the endpoints that are part of the same crud functionality